sdk/core/azure-core-v2/src/main/java/com/azure/v2/core/http/policy/BearerTokenAuthenticationPolicy.java

```diff
@@ -61,25 +71,36 @@ public BearerTokenAuthenticationPolicy(TokenCredential credential, String... sco
     }
 
     /**
-     * Synchronously executed before sending the initial request and authenticates the request.
+     * Executed before sending the initial request and authenticates the request.
      *
      * @param httpRequest The request context.
      */
-    public void authorizeRequestSync(HttpRequest httpRequest) {
-        setAuthorizationHeaderHelperSync(httpRequest, new TokenRequestContext().addScopes(scopes), false);
+    public void authorizeRequest(HttpRequest httpRequest) {
+        setAuthorizationHeaderHelper(httpRequest, new TokenRequestContext().addScopes(scopes).setCaeEnabled(true),
+            false);
     }
 
     /**
      * Handles the authentication challenge in the event a 401 response with a WWW-Authenticate authentication challenge
      * header is received after the initial request and returns appropriate {@link TokenRequestContext} to be used for
      * re-authentication.
+     * <p>
+     * The default implementation will attempt to handle Continuous Access Evaluation (CAE) challenges.
+     * </p>
      *
-     * @param httpRequest The request context.
+     * @param httpRequest The http request.
      * @param response The Http Response containing the authentication challenge header.
-     *
      * @return A boolean indicating if containing the {@link TokenRequestContext} for re-authentication
      */
-    public boolean authorizeRequestOnChallengeSync(HttpRequest httpRequest, Response<?> response) {
+    public boolean authorizeRequestOnChallenge(HttpRequest httpRequest, Response<?> response) {
+        if (AuthorizationChallengeParser.isCaeClaimsChallenge(response)) {
```

Comment:
Can this be in a util class, something like `AuthUtils.isCaeClaimsChallenge` ?
This and few others below.

Reply:
AuthUtils is made from a spec, this is not industry standard, moved it as a helper to the policy,